core/lib/globals_shared.h

```diff
@@ -1899,24 +1905,39 @@ typedef union _dr_simd_t {
 #    endif
 #    ifdef X64
 #        ifdef WINDOWS
-/*xmm0-5*/
+/*[xy]mm0-5*/
+#            define MCXT_NUM_SIMD_SSE_AVX_SLOTS                                 \
+                6 /**< Number of [xyz]mm reg slots in dr_mcontext_t pre AVX-512 \
+                   * in-use.                                                    \
```

Comment:
Move this to a whole-line before comment: no reason to squish it into a post-comment form.

Reply:
Done.